Follow up on varargs logging arguments

hubertp opened the pull request:
### Pull Request Description

Updated remaining logger calls to use varargs; some locations were skipped in #12661 and were showing up in logs incorrectly formatted.


### Checklist

- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.

hubertp committed:
Follow up on varargs logging arguments

Updated remaining logger calls to use varargs; some locations were
skipped in #12661 and were showing up in logs incorrectly formatted.

JaroslavTulach reviewed (approved):
Thank you for the additional fixes.

hubertp committed:
Merge branch 'develop' into wip/hubert/logger-args-followup

hubertp committed:
Fix formatting in TruffleLogger

```
[warn] WARNING: Initializing the context in a different working directory than the one containing the project root. This may lead to relative paths not behaving as advertised by `File.new`. Please run the engine inside of `{}` directory.
```
